### DISCUSSION TITLE
A Story about Test Coverage Metrics

### DISCUSSION THREAD

**Discussion by mauriciomenegaz** [therubyradar/submissions#58] [Submissions]
Why is it not enough to have 100% line coverage in your tests?
What should you be worrying instead?
Check it out on this article: https://coolrequest.dev/2022/05/24/a-story-about-test-metrics.html

**Comment by andrewmcodes** [comment]
Great article! I accidentally missed this when we were making the post this week but  keep submitting and we will get it in next time 😄
